cmd/ebrelayer/contract/Peggy.sol

```diff
@@ -102,6 +103,10 @@ interface IERC20 {
 contract ERC20 is IERC20 {
     using SafeMath for uint256;
 
+    string public name;
+    string public symbol;
```

Comment:
we may want to change this to `denom` to be consistent with the SDK. Thoughts?

Reply:
this is a specific variable in the erc20 spec so we ideally want to keep to that, and the contract is creating ethereum-specific events. I think the translation from symbol -> denom in the relayer makes sense/seems most appropriate. Also, related, see https://github.com/cosmos/peggy/issues/69